at-hoangtran/Week3/Day5/Exercise1-Day5.rb

```diff
@@ -0,0 +1,35 @@
+class Student
+  attr_accessor :first_name, :last_name, :list_of_courses
+  def initialize
+    @list_of_courses = []
+  end
+
+  def first_name
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/074932088ed5cad5e69983d076619cff8875320e/at-hoangtran/Week3/Day5/Exercise1-Day5.rb#L7"></span>Lint/DuplicateMethods: Method <code>Student#first_name</code> is defined at both at-hoangtran/Week3/Day5/Exercise1-Day5.rb:2 and at-hoangtran/Week3/Day5/Exercise1-Day5.rb:7.</td>
    </tr>
  </tbody>
</table>